manual/manual/cmds/unified-options.etex

```diff
@@ -745,7 +745,7 @@ that are currently defined are ignored. The warnings are as follows.
 \input{warnings-help.tex}
 \end{options}
 
-The default setting is "-w +a-4-6-7-9-27-29-32..39-41..42-44-45-48-50".
+The default setting is "-w +a-4-6-7-9-27-29-32..39-40-41..42-44-45-48-50-60".
```

Comment:
same here (even more confusing) and in `utils/warnings.ml`

Reply:
Ok. I had thought that the list delibrately only used a range for 32...39 because IIRC they are related warnings. I'll merge all the adjacent ones then.

Reply:
Fixed